backend/python/app/models/entity_mongo.py

```diff
@@ -0,0 +1,16 @@
+from mongoengine import *
+from .enum import enum
+
+
+class Entity(Document):
+    string_field = StringField(max_length=200, required=True)
+    int_field = IntField(required=True)
+    enum_field = StringField(max_length=200, required=True, choices=enum)
```

Comment:
I think it's ok to put the choices array directly in here (without import from `enum.py`)